ballsdex/packages/trade/menu.py

```diff
@@ -155,6 +156,7 @@ def __init__(
         self.task: asyncio.Task | None = None
         self.current_view: TradeView | ConfirmView = TradeView(self)
         self.message: discord.Message
+        self.end_time = math.ceil((datetime.now(timezone.utc) + timedelta(minutes=30)).timestamp())
```

Comment:
You can instead use this over having extra to import. It's a part of [discord.py](https://discordpy.readthedocs.io/en/stable/api.html#discord.utils.format_dt)
```suggestion
        self.end_time = discord.utils.format_dt(datetime.now(timezone.utc) + timedelta(minutes=30))
```